### PR TITLE
Add drop for arc enabled keyvault extension

### PR DESCRIPTION
**Pull request by george-markou**

### CODE REVIEW

File: drops/arc_server_windows_keyvault_extension.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "Type": "tutorial_guide",
+    "Difficulty": "Medium",
+    "ProgrammingLanguage": [
+      "PowerShell"
+    ],
+    "Products": [
+      "Arc",
+      "Arc-enabled servers"
+    ],
+    "LastModified": "",
```
Comment (fcabrera23) [inline drops/arc_server_windows_keyvault_extension.json:22]: Please add a date  - e.g: "2024-09-23T00:00:00.000Z" - Same for LastModified

Comment (george-markou) [reply drops/arc_server_windows_keyvault_extension.json]: Resolved, added modify and created date.

**Comment by george-markou** [comment]
@microsoft-github-policy-service agree

**Review by fcabrera23 — APPROVED** [review]
LGTM

**Pull request merged**